src/main/kotlin/no/nav/helse/prosessering/v1/PreprosseseringV1Service.kt

```diff
@@ -87,6 +99,45 @@ internal class PreprosseseringV1Service(
         )
     }
 
+    /**
+     * Slår opp barnets navn, gitt enten alternativId, fødselsNummer eller aktørId.
+     */
+    private suspend fun slaaOppBarnetsNavn(
+        barn: Barn,
+        correlationId: CorrelationId
+    ): String {
+
+        return when {
+            // Dersom barnet har navn, returner navnet.
+            !barn.navn.isNullOrBlank() -> barn.navn!!
+
+            // Ellers, hvis barnet har et fødselsNummer ...
+            !barn.fodselsnummer.isNullOrBlank() -> {
```

Comment:
fodselsnummer & alternativId -oppslagene på navn kan feile (at de har fått ID men register ikke er oppdatert)
Så vi bør håndtere ev feil på oppslagene i disse to scenarioene. Før aktør-ID så er det riktig å feile om oppslaget feiler, da må vi prøve igjen senere

Reply:
Det gjøres retry allerede i selve service metoden. Eller mener du at vi skal retry på `val barnetsNavn: String? = slaaOppBarnetsNavn(melding.barn, correlationId = correlationId)`?
```kotlin
val json = Retry.retry(
            operation = "hente-navn",
            initialDelay = Duration.ofMillis(200),
            factor = 2.0,
            logger = logger
        ) {
            val (request, _, result) = Operation.monitored(
                app = "pleiepengesoknad-prosessering",
                operation = "hente-navn",
                resultResolver = { 200 == it.second.statusCode }
            ) { httpRequest.awaitStringResponseResult() }

            result.fold(
                { success -> JSONObject(success) },
                { error ->
                    logger.error("Error response = '${error.response.body().asString("text/plain")}' fra '${request.url}'")
                    logger.error(error.toString())
                    throw IllegalStateException("Feil ved henting av navn.")
                }
            )
        }
```